web/src/main/java/org/mskcc/cbio/portal/web/ProxyController.java

```diff
@@ -54,7 +54,7 @@
   public void setBitlyURL(String property) { this.bitlyURL = property; }
 
   private String sessionServiceURL;
-  @Value("${session.service.url}")
+  @Value("${session.service.url:}")
```

Comment:
@ersinciftci The session service is a separate Spring Boot application that requires a Mongo database.  If you have it running and set the session.service.url in the portal application then the portal application uses the session service to store the information the portal needs for bookmarking.  Most people will not have the session service running, so it should be turned off by default.